src/main/interface/cli.c

```diff
@@ -936,12 +977,28 @@ static void cliSerialPassthrough(char *cmdline)
             if (strstr(tok, "tx") || strstr(tok, "TX"))
                 mode |= MODE_TX;
             break;
+        case 3:
+        	// When programming Arduino based devices such as MinimOSD, the DTR line is used to control
+        	// the reset. This parameter is the pin name, for example C8. This allows
+        	// any GPIO to be used for driving DTR.
+        	if (strToPin(tok, &serialPassthroughDtrTag)) {
+                if (serialPassthroughDtrTag == IO_TAG_NONE) {
+                	cliPrintLine("Invalid DTR pin");
+                	return;
+                } else {
+                	serialPassthroughDtrPin = IOGetByTag(serialPassthroughDtrTag);
```

Comment:
`IOGetByTag` does check if pin is available, but will correctly accept `IO_TAG_NONE` - it will validate `strToPin`

```
if (!strToPin(tok, &serialPassthroughDtrTag) || !IOGetByTag(serialPassthroughDtrTag)) {